explorer/src/bin/clnodes.ts

```diff
@@ -36,16 +36,21 @@ const bootstrap = async (cb: any) => {
 const _ = yargs
   .usage('Usage: $0 <command> [options]')
   .command({
-    command: 'add <name>',
+    command: 'add <name> [url]',
     aliases: 'create',
     describe: 'Add a chainlink node',
     builder: (yargs): any => {
-      yargs.positional('name', {
-        describe: 'The name of the Chainlink Node to create',
-        type: 'string'
-      })
+      yargs
+        .positional('name', {
+          describe: 'The name of the Chainlink Node to create',
+          type: 'string'
+        })
+        .describe('u', 'The URL of the Chainlink Node to create')
+        .alias('u', 'url')
+        .default('u', '')
```

Comment:
Does this need the default if it saves a node without a URL with a `NULL` instead of empty string?

I'd prefer to be consistent for existing records + new records that don't have a website